feeds/backend.php

```diff
@@ -30,21 +33,45 @@
         switch($content) {
             case "posted":
                 $condition = sprintf("state='%s'", PROJ_SUBMIT_PG_POSTED);
+                $desc = sprintf(_("The latest releases posted to Project Gutenberg from %1\$s."), $site_name);
                 break;
             case "postprocessing":
                 $condition = sprintf("state='%s'", PROJ_POST_FIRST_AVAILABLE);
+                $desc = sprintf(_("The latest releases available at %1\$s for post-processing."), $site_name);
                 break;
             case "proofing":
                 $condition = sprintf("state='%s'", PROJ_P1_AVAILABLE);
+                $desc = sprintf(_("The latest releases available at %1\$s for proofreading."), $site_name);
                 break;
             case "smoothreading":
-                $condition = "
-                    state = 'proj_post_first_checked_out' AND
-                    smoothread_deadline > UNIX_TIMESTAMP()";
+                $query = "
+                    SELECT 
+                        projectid, nameofwork, genre, language, FROM_UNIXTIME(e.timestamp)
+                    FROM projects
+                    JOIN project_events e USING (projectid)
+                    WHERE
+                        e.event_type = 'smooth-reading' AND
+                        e.details1 = 'text available' AND
+                        e.timestamp > UNIX_TIMESTAMP() - (30*24*60*60)
```

Comment:
Might add a comment above this query that we're only looking at SR projects that have moved into SR in the last 30 days -- hence this bit.

Reply:
Added in 9ec53768b5bc3af587162cebb76e68cc2d1b508c.